openmls/src/group/tests/test_proposal_validation.rs

```diff
@@ -1105,12 +1145,12 @@ fn test_valsem106(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider
                     let result = alice_group.add_members(
                         backend,
                         &alice_credential_with_key_and_signer.signer,
-                        &[test_kp.clone()],
+                        &[test_kp_2.clone()],
```

Comment:
I'm not sure I understand why we need this.
The only difference between this test and the one above is that it's by value instead of reference. But this shouldn't influence the test.
Can you ellaborate why you changed this?

Reply:
We iterate over `[ProposalInclusion::ByReference, ProposalInclusion::ByValue]` and thus call `propose_add_member` and `add_members` with the same key package(s). Thus, instead of testing what the test says it does, we got an error due to duplicate signature keys.

Reply:
Ah ok. I guess the idea was to have these on fresh groups instead or something. But this works too.